tensilelite/rocisa/cmake/RocisaConfig.cmake

```diff
@@ -0,0 +1,74 @@
+################################################################################
+#
+# Copyright (C) 2022-2025 Advanced Micro Devices, Inc. All rights reserved.
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+#
+################################################################################
+
+include(CMakeParseArguments)
+if(NOT DEFINED Rocisa_ROOT)
+    # Compute the installation prefix relative to this file.
+    get_filename_component(Rocisa_PREFIX "${CMAKE_CURRENT_LIST_FILE}" PATH)
+    get_filename_component(Rocisa_PREFIX "${Rocisa_PREFIX}" PATH)
+    if (WIN32)
```

Comment:
Just change it to `"${Rocisa_PREFIX}/bin/RocisaGetPath${CMAKE_EXECUTABLE_SUFFIX}" and remove the redundant branch and code.